editor.planx.uk/src/@planx/components/shared/Preview/SummaryList.tsx

```diff
@@ -229,42 +211,72 @@ function SummaryListsBySections(props: SummaryListsBySectionsProps) {
   const { trackBackwardsNavigation } = useAnalyticsTracking();
+  const [isDialogOpen, setIsDialogOpen] = useState(false);
+  const [nodeToChange, setNodeToChange] = useState<Store.nodeId | undefined>(
+    undefined,
+  );
+
+  const handleCloseDialog = (isConfirmed: boolean) => {
+    setIsDialogOpen(false);
+    if (isConfirmed && nodeToChange) {
+      trackBackwardsNavigation("change", nodeToChange);
+      props.changeAnswer(nodeToChange);
+    }
+  };
 
-  const handleChangeAnswer = (id: string) => {
-    trackBackwardsNavigation("change", id);
-    props.changeAnswer(id);
+  const handleChange = (nodeId: Store.nodeId) => {
+    setNodeToChange(nodeId);
+    setIsDialogOpen(true);
   };
 
   return (
-    <Grid showChangeButton={props.showChangeButton}>
-      {props.summaryBreadcrumbs.map(
-        ({ component: Component, nodeId, node, userData }, i) => (
-          <React.Fragment key={i}>
-            <Component
-              nodeId={nodeId}
-              node={node}
-              userData={userData}
-              flow={props.flow}
-              passport={props.passport}
-            />
-            {props.showChangeButton && (
-              <dd>
-                <Link
-                  onClick={() => handleChangeAnswer(nodeId)}
-                  component="button"
-                  fontSize="body2.fontSize"
-                >
-                  Change
-                  <span style={visuallyHidden}>
-                    {node.data?.title || node.data?.text || "this answer"}
-                  </span>
-                </Link>
-              </dd>
-            )}
-          </React.Fragment>
-        ),
-      )}
-    </Grid>
+    <>
+      <Grid showChangeButton={props.showChangeButton}>
+        {props.summaryBreadcrumbs.map(
+          ({ component: Component, nodeId, node, userData }, i) => (
+            <React.Fragment key={i}>
+              <Component
+                nodeId={nodeId}
+                node={node}
+                userData={userData}
+                flow={props.flow}
+                passport={props.passport}
+              />
+              {props.showChangeButton && (
+                <dd>
+                  <Link
+                    onClick={() => handleChange(nodeId)}
+                    component="button"
+                    fontSize="body2.fontSize"
+                  >
+                    Change
+                    <span style={visuallyHidden}>
+                      {node.data?.title || node.data?.text || "this answer"}
+                    </span>
+                  </Link>
+                </dd>
+              )}
+            </React.Fragment>
+          ),
+        )}
+      </Grid>
+      <ConfirmationDialog
+        open={isDialogOpen}
+        onClose={handleCloseDialog}
+        title="Confirm"
+        confirmText="Yes"
+        cancelText="No"
+      >
+        <Typography>
+          If you change this answer, you’ll need confirm all the other answers
```

Comment:
typo: `...you'll need _to_ confirm...`

also personal preference: i'd keep it very short & simple & delete the second "because" sentence !

Reply:
Copy/pasted content from Trello - didn't check for typos 🤦 

Agreed that for me the shorter version is clearer / more meaningful. As this is content from Nomensa I'll pass it forward as is (with typo fixed!) but raise this at PO testing 👍